### ISSUE TITLE
Android: Irrlicht patch has wrong line endings

### ISSUE THREAD

**Issue by stujones11** [minetest/minetest#6750]
Since 2507d32afe05c73bb27ed221c2a592b0894fdc68 the included [irrlicht-native_activity.patch](https://github.com/minetest/minetest/commit/2507d32afe05c73bb27ed221c2a592b0894fdc68#diff-daa8d52a16bcc47ecc3d0db26d566e6c) has the wrong line endings and now fails to apply. I am not sure if I did something wrong here yet the same patch works fine when pulled from the original PR's feature branch of my own repo??

I'd make another PR myself, though I fear the same thing would happen again. Does anyone have any idea what could have happened here?

This is a diff of the diff, the latter part being the correct version (copied from console)
```
 @@ -8,6 +8,9 @@
- 
- #include "irrString.h"
- #include "os.h"
-+#if defined(_IRR_COMPILE_WITH_ANDROID_DEVICE_)
-+#include <android/native_activity.h>
-+#endif
- 
- namespace irr
- {
+ ^M
+ #include "irrString.h"^M
+ #include "os.h"^M
++#if defined(_IRR_COMPILE_WITH_ANDROID_DEVICE_)^M
++#include <android/native_activity.h>^M
++#endif^M
+ ^M
+ namespace irr^M
+ {^M
```

**Comment by stujones11** [comment]
Unfortunately this issue still remains even though I fixed the line-endings in 22a891a925d68a096420d3383a2e31b58e3eb244

Could this be a problem with .gitignore?

**Comment by nerzhul** [comment]
Fixed on stable 0.4 https://github.com/minetest/minetest/commit/0ef9c53a8c20872ce69f1c539167497a04efe529#diff-daa8d52a16bcc47ecc3d0db26d566e6c
